lib/HTFeed/StorageZipAudit.pm

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/perl
+package HTFeed::StorageZipAudit;
+
+use strict;
+use warnings;
+use Carp;
+use File::Temp;
+use FindBin;
+use Log::Log4perl qw(get_logger);
+
+use HTFeed::Config qw(get_config);
+use HTFeed::DBTools qw(get_dbh);
+use HTFeed::Volume;
+
+sub new {
+  my $class        = shift;
+  my $storage_name = shift;
+
+  if (defined $storage_name) {
+    # Instantiate audit subclass based on storage class name.
+    my $config = get_config('storage_classes');
+    my $storage_config = $config->{$storage_name};
+    die "unable to get config for $storage_name" unless $storage_config;
+
+    my @components = split '::', $storage_config->{class};
```

Comment:
My first thought here was that this is a little bit hairy, and it might be preferable to explicitly specify in the storage config what audit class to use. However, it also seems unlikely that we would want to vary the audit class independently of the storage class. Perhaps we could instead add a method on the storage instance that returns a zip auditor for that storage? Then we would start the audit as something like `HTFeed::Storage::storage_for('instance')->zip_auditor->run()`

Reply:
That would also allow us to get rid of the empty `HTFeed::StorageZipAudit::PrefixedVersions` subclass

Reply:
I think this change should be high priority